DEVELOPING.md

```diff
@@ -149,3 +149,39 @@ This should create a build job, and then dispatch it to the build worker.
 You can view the build progress in the web UI or by viewing `/hab/svc/builder-worker/data/876066265100378112/log_pipe-876066265100378112.log`. Replace `876066265100378112` with the group ID output by the `start` command.
 
 Note: you will need to upload additional packages to the core origin for the `core/nginx` build to succeed. Follow the same procedure as for `core/hab-backline`. Currently `core/gcc` and `core/libedit` are required.
+
+
+#### Building Dependencies From Live for Upload into Your Local Depot
+
+In the likely scenario that your build has dependencies not initially uploaded to your local core origin, you may run into an error similar to the following:
+
+```
+» Installing core/node
+☁ Determining latest version of core/node in the 'stable' channel
+✗✗✗
+✗✗✗ Package not found. If you are attempting to install from a local depot with an upstream configured, try again in a few seconds.
+✗✗✗
+```
+
+One way to resolve this dependency is by swapping your Builder URL, building locally, and then uploading to your local depot.
```

Comment:
This seems pretty painful - why would we do that instead of just leveraging the upstream already built packages?